front/src/modules/ui/navbar/components/NavCollapseButton.tsx

```diff
@@ -29,15 +32,27 @@ const StyledCollapseButton = styled.button`
   padding: 0;
 
   user-select: none;
+  visibility: ${({ isHovered }) => (isHovered ? 'visible' : 'hidden')};
```

Comment:
```suggestion
  visibility: ${({ hide }) => (hide ? 'hidden' : 'visible')};
```

Reply:
It will reverse the functionality the same as above like `NavCollapseButton` will hidden when the mouse is hovered over the nav bar which leads to not being able to toggle at nav bar and visible at mouse leaves.